### PR TITLE
fix Issue #445

### PR DESCRIPTION
**Pull request by Lilyhanig**

### CODE REVIEW

File: src/core/simulation_results.jl [hunk 1]
```diff
@@ -15,7 +15,7 @@ function SimulationResultsReference(sim::Simulation; kwargs...)
         interval = get_stage_interval(sim, stage_name)
         resolution = PSY.get_forecasts_resolution(get_sys(stage))
         chronologies["stage-$stage_name"] = convert(Int, (interval / resolution))
-        base_powers[stage_name] = PSY.get_base_power(sim.stages[stage_name].sys)
+        base_powers[stage_name] = PSY.get_basepower(sim.stages[stage_name].sys)
```
Comment (daniel-thom) [inline src/core/simulation_results.jl:18]: Isn't this incorrect?

Comment (jd-lara) [reply src/core/simulation_results.jl:18]: Yes, it should be `get_base_power()`

File: src/core/simulation_results.jl [hunk 2]
```diff
@@ -274,7 +274,10 @@ function load_simulation_results(
     variable::Array;
     kwargs...,
 )
-    results_folder = sim_output.results_folder
+    results_folder = joinpath(sim_output.results_folder, stage_name)
+    if !isdir(results_folder)
```
Comment (daniel-thom) [inline src/core/simulation_results.jl:278]: If you use `mkpath` then you can avoid checking for existence.  Just an FYI if prefer conciseness.

File: src/core/simulation_results.jl [hunk 3]
```diff
@@ -564,17 +570,61 @@ variable = get_result_variable(results, :ON, ThermalStandard)
...
-    return IS.get_variables(results)[variable_name]
-end
-
-function get_variable_names(results::IS.Results)
-    return collect(keys(results.variable_values))
+function load_results(folder_path::String)
+    if isfile(folder_path)
+        throw(ArgumentError("Not a folder path."))
+    end
+    files_in_folder = collect(readdir(folder_path))
```
Comment (daniel-thom) [inline src/core/simulation_results.jl:584]: readdir already returns an Array.  Several instances are in the PR.

File: src/core/simulation_results.jl [hunk 4]
```diff
@@ -564,17 +570,61 @@ variable = get_result_variable(results, :ON, ThermalStandard)
...
+    vars_result = Dict{Symbol, DataFrames.DataFrame}()
+    dual_result = Dict{Symbol, Any}()
+    dual_names = _find_duals(variable_list)
+    param_names = _find_params(variable_list)
+    variable_list = setdiff(variable_list, vcat(dual_names, param_names, ".DS_Store"))
+    param_values = Dict{Symbol, DataFrames.DataFrame}()
+    for name in variable_list
+        variable_name = splitext(name)[1]
+        file_path = joinpath(folder_path, name)
+        @show file_path
```
Comment (daniel-thom) [inline src/core/simulation_results.jl:598]: stale debug print

**Pull request merged**